README.md

```diff
@@ -13,7 +13,7 @@ To run locally:
     ./get-content.sh
     ```
 
-3. Build and serve locally.
+3. Build and serve locally. You will need [bundler](https://bundler.io/). You can install it with `gem install bundler` and then can run `bundle install` before running this bundle exec command below:
```

Comment:
Hi @danigrant, thank you for the PR>

These instructions assume the user has already installed Jekyll properly.  So, this sub-section implies that isn't the case.  Would you like to instead like to add an intro sentence pointing users to the the Jekyll install if they haven't done that bit?